### PR TITLE
fix error message #2198

### LINKED ISSUE #2198: Make error message user friendly.
**Describe the bug**
on setting ConstVal to except 1/0, the Contention error of NaN and NaN at ConstVal shows up.

**To Reproduce**
1. Go to '[simulator](https://circuitverse.org/simulator)'
2. Click on 'select a ConstVal input.
3. Double click the Const Val input and set value except 1/0. 
4. See an error.

**Expected behavior**
A user-friendly error message like, "ConstVal set to Non-binary value."

**Screenshots**

image

**Are you working on this issue? (Yes/No)**
No.

### PR DESCRIPTION
**Pull request by Jayita10**
Fixes #2198 

### Screenshots of the changes (If any) [-]
Screenshot (252)

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

File: simulator/src/node.js [hunk 1]
```diff
@@ -387,7 +387,7 @@ export default class Node {
                     node.highlighted = true;
                     var circuitName = node.scope.name;
                     var circuitElementName = node.parent.objectType;
-                    showError(`Contention Error: ${this.value} and ${node.value} at ${circuitElementName} in ${circuitName}`);
+                    showError(`ConstVal set to Non-binary value.`);
```
[the remark this answers is not in the document]
Comment (Jayita10) [reply simulator/src/node.js:390]: @ShreyaPrasad1209 please help me to pass the all test cases. Thank you

**Review by ShreyaPrasad1209 — CHANGES_REQUESTED** [review]
@Jayita10  can you attach screenshots of your changes? Also, there's no need to write "sign off: your name" in the PR description.

**Comment by Jayita10** [comment]
> @Jayita10 can you attach screenshots of your changes? Also, there's no need to write "sign off: your name" in the PR description.

Please review :)

**Review by ShreyaPrasad1209 — APPROVED** [review]
Great work @Jayita10 🎉

**Comment by tachyons** [comment]
@satu0king

**Comment by gr455** [comment]
> This breaks contention message for other elements also :(

@Jayita10 I am reverting this commit because this breaks the error message for other elements too. Let me know if you would like to follow this issue up. Thanks!

**Comment by Jayita10** [comment]
> > This breaks contention message for other elements also :(
> 
> @Jayita10 I am reverting this commit because this breaks the error message for other elements too. Let me know if you would like to follow this issue up. Thanks!

@gr455 I'm working on this issue

**Comment by gr455** [comment]
> > > This breaks contention message for other elements also :(
> > 
> > @Jayita10 I am reverting this commit because this breaks the error message for other elements too. Let me know if you would like to follow this issue up. Thanks!
> 
> @gr455 Okay, I'm working on this issue

Amazing!
I think that it's better to add validation in ConstVal to not accept non-binary data instead of changing the error message because changing the error message for a single element is not easy

**Pull request merged**